PatientRegistrationApp/src/components/HouseholdContactInfo.vue

```diff
@@ -119,21 +119,33 @@ import EventBus from '../eventBus'
 		verifyFormContents()
 		{
 			var valid = true
-			var message = "Woops! You need to enter the following fields:"
+			var message = "Woops! You need to enter the following field(s):"
```

Comment:
The error/warning for the greeting page is no longer there! This doesn't have to do with the current selected code here as well.

Reply:
Edit: after some more testing, I realize the warning only appears if you've selected a workflow but not the language. We should enable it if they just try to move forward without selecting anything, that the warning pops up. Can you write a card for this?